lab_4_fill_words_by_ngrams/main.py

```diff
@@ -56,6 +74,17 @@ def _postprocess_decoded_text(self, decoded_corpus: tuple[str, ...]) -> str:  #
         Raises:
             ValueError: In case of inappropriate type input argument or if input argument is empty.
         """
+        if not isinstance(decoded_corpus, tuple) or not decoded_corpus:
+            raise ValueError
+        decoded_corpus = ' '.join(decoded_corpus).replace(f' {self._end_of_word_token}', '.').capitalize()
+        capitalized_text = decoded_corpus.split('. ')
+        for i in range(1, len(capitalized_text)):
+            capitalized_text[i] = capitalized_text[i][0].upper() + capitalized_text[i][1:]
+        final_text = '. '.join(capitalized_text)
+        if final_text.endswith("."):
+            return final_text
+        else:
+            return final_text + '.'
```

Comment:
can you reduce number of return statements to one?

Reply:
fixed